src/main/kotlin/com/avail/descriptor/bundles/MessageBundleDescriptor.kt

```diff
@@ -335,7 +331,7 @@ class MessageBundleDescriptor private constructor(
 		 * @param splitter
 		 *   A MessageSplitter for this message name.
 		 * @return
-		 *   A new [message&#32; bundle][A_Bundle].
+		 *   A new [message bundle][A_Bundle].
```

Comment:
`&#32;` (but without space).

Reply:
I also fixed a busted instance of this in the class comment.